ansible/playbooks/roles/repository/tasks/RedHat/install-packages.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- include_vars:
+    file: roles/common/vars/main.yml
+    name: common_vars
+
+# Fix to solve issues with expiring certificates on Azure
+# https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/redhat/redhat-rhui#update-expired-rhui-client-certificate-on-a-vm
+# TODO: Needs to be updated for RHEL 8
+- name: Update repository certificates
+  command: yum update -y --disablerepo='*' --enablerepo='*microsoft*'
+  when:
+    - common_vars.provider == "azure" and ansible_distribution_major_version == "7"
```

Comment:
add `ansible_os_family` check, use the list of conditions:

```yaml
when:
 - condition0
 - condition1
 - condition2
```

Reply:
Done.